### PR TITLE
DKMS added

### PR DESCRIPTION
**Pull request by dominatornsk**

### CODE REVIEW

File: init.d/dkms [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/openrc-run
+
+. /etc/default/dkms
+
+# list dkms modules for the current running kernel
+list_dkms_modules() {
+  shopt -s nullglob
+  declare -a DKMS_MOULES
```
Comment (notfoss) [inline init.d/dkms]: Should be DKMS_MODULES.

File: init.d/dkms [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  # check modules are unloaded
+  # sometimes modprobe -r fail but modules are removed
+  for m in "${DKMS_MODULES[@]}"; do
+    [[ -e /sys/modules/$m ]] && return 1
+  done
+  return 0
+}
+
+start() {
+    ebegin "DKMS wird gestartet"
```
Comment (notfoss) [inline init.d/dkms]: Please use english for the message.

**Comment by notfoss** [comment]
Thanks for the patch. I have made a couple of comments on the script, please fix those. Also, please use 4 spaces for indentation.

**Comment by dominatornsk** [comment]
[DE]Ich habe Fehler korrigiert
[EN]I fixed errors

**Comment by notfoss** [comment]
Thanks. Merged.

**Pull request merged**